release_process.md

```diff
@@ -1,3 +1,57 @@
 # Release Process
 
-<!-- TBD -->
\ No newline at end of file
+This document outlines the release process for Ignite. It ensures consistency, quality, and clear communication with users.
+Ignite uses [semantic versioning](https://semver.org/) to indicate the stability and compatibility of releases.
+
+## Release Branches
+
+Releases are tagged from release branches. The release branch is typically named after the release version, such as `release/v28.x.y` or `release/v30.x.y`. The release branch is created from the `main` branch and contains all the changes that will be included in the release.
+
+## Development Branch
+
+The `main` branch is the development branch for Ignite. All new features and bug fixes are developed on this branch. The `main` branch is typically updated daily or weekly, depending on the amount of development activity.
+
+## Backporting Features and Bug Fixes
+
+Features or bug fixes that are ready to be included in a release must be backported from the `main` branch to the release branch. This is done using Mergify, a CI/CD tool that automates the process of backporting changes. Add the `backport release/x.y.z` label to a pull request (PR) to indicate that it should be backported to the release branch. Mergify will automatically backport the PR when it is merged into the `main` branch.
+
+## All PRs Target Main
+
+All pull requests (PRs) should target the `main` branch. This ensures that all changes are always integrated into the development branch and are eligible for backporting to release branches.
+
+## Changelog
+
+The changelog is a file that lists all of the changes that have been made to Ignite since the last release. The changelog must be up to date before a release is made.
```

Comment:
```suggestion
The changelog lists all of the changes that have been made to Ignite since the last release. The changelog must be up-to-date before a release is made.
```

Reply:
I believe we can just include this point as part of changelog point in the next section ("Release Preparation and Testing"), and link to the changelog.